app/src/main/java/com/example/navadon/androidonclickevent/Option0Activity.java

```diff
@@ -39,13 +42,23 @@ public void process(View v) {
     }
 
     // To greet the user
-    private void greet(){
+    private void greet() {
         tvOutput.setText(getString(R.string.greeting) + " " + etInput.getText().toString());
     }
 
     // To hide Android soft keyboard
-    private void hideKeyboardInput(View v){
+    private void hideKeyboardInput(View v) {
         InputMethodManager imm = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
         imm.hideSoftInputFromWindow(v.getWindowToken(), 0);
     }
+
+    public void onNextButtonClick(View view) {
+        intent = new Intent(this, Option1Activity.class);
```

Comment:
You can actually put this into "process(View v)" then check if that "v" equals Next or Back button instead of having 2 individual methods like this.